app/javascript/stylesheets/application.scss

```diff
@@ -30,6 +30,16 @@ body{
   background-color: #556588;
 }
 
+.btn-submit {
+  color: whitesmoke;
+  background-color: #798db6;
+}
+
+.btn-submit:hover {
+  color: whitesmoke;
+  background-color: #556588;
```

Comment:
In 30 line is  the same property with the same value. Pleasebrrad about scss constants, and DRY i

Reply:
fix it